src/logic/operate.js

```diff
@@ -0,0 +1,16 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operator) {
+  const operatorsMap = {
+    '÷': Big.prototype.div,
+    '*': Big.prototype.times,
+    '+': Big.prototype.plus,
+    '-': Big.prototype.minus,
+    '%': Big.prototype.mod,
+  };
```

Comment:
The `%` operator should be turning the number into a percentage based number, e.g. 95 % == .95 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png))

Reply:
Oh, Thank you. Fixed.